controllers/src/test/scala/cromwell/pipeline/controller/AuthControllerTest.scala

```diff
@@ -20,31 +22,28 @@ class AuthControllerTest extends WordSpec with Matchers with MockFactory with Sc
   private val accessToken = "access-token"
   private val refreshToken = "refresh-token"
   private val accessTokenExpiration = 300
+  private val password = TestUserUtils.userPassword
 
   "AuthController" when {
 
     "signIn" should {
 
-      "return token headers if user exists" taggedAs Controller in {
-        val email = "email@cromwell.com"
-        val password = "password"
+      "return token headers if user exists" in {
+        val signInRequest = SignInRequest(UserEmail("email@cromwell.com"), password)
         val authResponse = AuthResponse(accessToken, refreshToken, accessTokenExpiration)
-        val signInRequestStr = s"""{"email":"${email}","password":"${password}"}"""
-        val httpEntity = HttpEntity(`application/json`, signInRequestStr)
-        (authService.signIn _ when SignInRequest(email, password)).returns(Future(Option(authResponse)))
+        val httpEntity = HttpEntity(`application/json`, Json.stringify(Json.toJson(signInRequest)))
+        (authService.signIn _ when signInRequest).returns(Future(Option(authResponse)))
```

Comment:
maybe Future.successfull is relevant here

Reply:
I don't think so, because signIn method expects an Option type